### PR TITLE
SNOW-1429890: Adding template for a SPCS basic app

### PR DESCRIPTION
**Pull request by sfc-gh-osalazarlizano**

### CODE REVIEW

File: spcs-basic/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+```sh
+export SNOWFLAKE_DEFAULT_CONNECTION_NAME=<your connection name>
+```
+
+## Create image repository, build and push our local service image
+
+The [service/](service/) directory contains a [Dockerfile](service/Dockerfile) that builds a
+simple Python server that responds to GET health checks, a GET for `/index.html`, as well as
+POSTing to `/echo` in the Snowflake External Function payload format. You can build it and
+push it to an image repository in your count like so:
```
Comment (sfc-gh-ralfaroviquez) [inline spcs-basic/README.md]:
```suggestion
push it to an image repository in your container like so:
```

File: spcs-basic/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Set your SnowCLI connection (optional)
+
+We use your default connection to connect to Snowflake and deploy the images / app. Set your
+default connection in your `config.toml` or by exporting the following environment variable:
+
+```sh
+export SNOWFLAKE_DEFAULT_CONNECTION_NAME=<your connection name>
+```
+
+## Create image repository, build and push our local service image
```
Comment (sfc-gh-ralfaroviquez) [inline spcs-basic/README.md]:
```suggestion
## Create image repository, build and push your local service image
```

File: spcs-basic/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Click on the `Grant` button to proceed.
+
+## Activate the application
+
+Once privileges are granted, a new `Activate` button should appear. Click the button and wait until the application is fully activated.
+The `Activate` button invokes the `grant_callback` defined in the [manifest.yml](app/manifest.yml) file, which then creates the `COMPUTE POOL` and `SERVICE` needed to launch the application.
+
+## Launch the application
+
+Once all services and pools are created, you are able to launch the app by clicking on the `Launch App` button. A new window will be displayed with the URL provided by the `Service` and `Endpoint` defined in the `default_web_endpoint` in the [manifest.yml](app/manifest.yml).
```
Comment (sfc-gh-ralfaroviquez) [inline spcs-basic/README.md]:
```suggestion
Once all services and pools are created, you will be able to launch the app by clicking on the `Launch App` button. A new window will be displayed with the URL provided by the `Service` and `Endpoint` defined in the `default_web_endpoint` in the [manifest.yml](app/manifest.yml).
```

**Review by sfc-gh-ralfaroviquez — APPROVED** [review]

File: spcs-basic/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Instructions
+
+## Set your SnowCLI connection (optional)
```
Comment (sfc-gh-cgorrie) [inline spcs-basic/README.md:11]: Let's add a section at the top for requirements (docker / docker buildx for example)

Comment (sfc-gh-osalazarlizano) [reply spcs-basic/README.md:11]: Done!

**Review by sfc-gh-cgorrie — APPROVED** [review]

**Pull request merged**